docs/src/lighting.md

```diff
@@ -0,0 +1,29 @@
+# Lighting
+
+For 3-D scenes, GLMakie offers several attributes to control the lighting of the scene.  These are set per Scene.
+
+- `ambient::Vec3f0`: The intensity of the ambient lighting.  Each element of the vector represents the intensity of color in R, G or B respectively.
+- `diffuse::Vec3f0`: The intensity of the diffuse lighting.  Each element of the vector represents the intensity of color in R, G or B respectively.
+- `specular::Vec3f0`: The intensity of the specular lighting.  Each element of the vector represents the intensity of color in R, G or B respectively.
```

Comment:
Maybe we should explain a bit more what `ambient`, `diffuse` and `specular` mean? Or refer to https://learnopengl.com/Lighting/Basic-Lighting?